tests/spec/unit/lib/Api.spec.js

```diff
@@ -95,6 +95,15 @@ describe('Api class', () => {
             expect(api.root).toEqual(testProjectDir);
             expect(api.locations).toEqual(jasmine.objectContaining(mockExpectedLocations));
         });
+
+        it('should throw error when the constructor is missing platformRootDir argument.', () => {
+            expect(() => new Api()).toThrowError(CordovaError, 'The path to the platform root directory was undefined or invalid.');
```

Comment:
Prettier would reformat like this:

```suggestion
            expect(() => new Api()).toThrowError(
              CordovaError,
              "The path to the platform root directory was undefined or invalid."
            );
```

(<https://prettier.io/playground/>)

Reply:
I applied your suggestions manually.

Also the settings you used on the prettier seemed to be all default. It uses double quotes when we use single quotes.
I went ahead made sure it was single quotes during the manual change.

I tend to keep things on one line, but i'll try to be more cautious on the length of the line.

Thanks for the review.